tensorflow/python/kernel_tests/extract_image_patches_op_test.py

```diff
@@ -125,5 +125,20 @@ def testKsize2x2Stride1x1Rate2x2Valid(self):
         padding="VALID",
         patches=patches)
 
+  def testComplexDataTypes(self):
+    """Test for complex data types"""
+    for dtype in [np.complex64, np.complex128]:
+      image = np.reshape(range(120), [2, 3, 4, 5]).astype(dtype)
+      patches = np.reshape(range(120), [2, 3, 4, 5]).astype(dtype)
+      for padding in ["VALID", "SAME"]:
```

Comment:
Please set a non-zero imaginary part for complex data.

Reply:
Thanks @rmlarsen. The PR has been updated.